jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpCookie.java

```diff
@@ -137,7 +146,14 @@ public HttpCookie(String setCookie)
         _version = cookie.getVersion();
         _expiration = _maxAge < 0 ? -1 : System.nanoTime() + TimeUnit.SECONDS.toNanos(_maxAge);
         // support for SameSite values has not yet been added to java.net.HttpCookie
-        _sameSite = getSameSiteFromComment(cookie.getComment());
+        SameSite sameSite = getSameSiteFromComment(cookie.getComment());
```

Comment:
Can we add javadoc to the constructor saying that comments will be checked for samesite.